components/Footer/Form/content/labelTexts.ts

```diff
@@ -0,0 +1,35 @@
+interface LabelTextsType {
+  nameInputPlaceHolder: string,
+  nameInputError: string,
+
+  emailInputPlaceHolder: string,
+  emailInputError: string,
+
+  phoneNumberPlaceHolder: string,
+  serviceSelectionPlaceHolder: string,
+
+  messagePlaceHolder: string,
```

Comment:
Is it intentional that there is no validation for phone/service ? You could create a type that wraps the placeholder and error, so it would look like this

`
Input: {
  placeholder: string:
  error: string;
}

LabelTextsType: {
  name: Input;
  email: Input;
  ...
}`